templates/bootstrap/pyproject.toml.j2

```diff
@@ -60,3 +60,20 @@ exclude = '''
 )/
 '''
 {% endif %}
+
+{% if check_manifest %}
+[tool.check-manifest]
+ignore = [
+    ".bumpversion.cfg",
+    "CHANGES/**",
```

Comment:
Is this check run when doing a release? At that point, CHANGES should actually be empty.

Reply:
It's run as part of our test suite so it's run against PRs and releases.

It's technically not empty at release time as there's still a `.TEMPLATE.rst` file in it.